cardano-cli/src/Cardano/CLI/Read.hs

```diff
@@ -789,6 +792,16 @@ readVotingProceduresFile w fp =
   conwayEraOnwardsConstraints w
     $ first VoteErrorFile <$> readFileTextEnvelope AsVotingProcedures fp
 
+voteTextSourceToText :: ()
+  => VoteTextSource
+  -> ExceptT VoteError IO Text
+voteTextSourceToText voteTextSource = do
```

Comment:
Call this function `readVoteAnchorTextSource` to make it more obvious we're _potentially_ doing IO and not just a conversion.